modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -6,6 +6,26 @@
 Users can be created and managed using a set of Cypher administration commands executed against the `system` database.
 When connected to the DBMS over `bolt`, administration commands are automatically routed to the `system` database.
 
+== User States
```

Comment:
I started adding comments but then decided to add the whole suggestion in a comment as otherwise, it might become too patchy. So here it is:

```
== User states

There are two types of user states in the `system` database:

* `ACTIVE` state (the default for new users) -- users can log into Neo4j and perform queries according to their privileges.

* `SUSPENDED` state label:enterprise-edition[] -- users are not permitted to log into Neo4j using the credentials in the system graph. 
If they attempt to do so, they will lose all their assigned roles, including `PUBLIC`.
+
[NOTE]
====
Users with roles assigned by an external authorization provider can still access Neo4j by logging in with their external authentication credentials.
To prevent this, you must suspend their access by using the mechanisms provided by the identity system.
It is important to note that even if access is suspended, their roles will remain assigned, including the `PUBLIC` role.
====
```

Reply:
> users are not permitted to log into Neo4j using the credentials in the system graph. 
If they attempt to do so, they will lose all their assigned roles, including `PUBLIC`.

Isn't really the case though. They can't log in at all if they are suspended, that part has nothing to do with the roles. If they already have an open connection we don't kick them out when they get suspended but it is in this open connection their roles will be removed (temporarily, if they get reactivated then the roles will still be there).

Reply:
Ok, let me think again.

Reply:
Is it correct now?

````
== User states

There are two types of user states in the `system` database:

* `ACTIVE` state (the default for new users) -- users can log into Neo4j and perform queries according to their privileges.

* `SUSPENDED` state label:enterprise-edition[] -- users cannot log into Neo4j using the credentials in the system graph. 
If the users are suspended while using Neo4j, they lose all assigned roles, including `PUBLIC`, until reactivated.
+
[NOTE]
====
Users with external authentication and roles assigned by an external authorization provider must be suspended using the mechanisms of their identity provider. 
====

Reply:
Ah, yes. I might have missed that.

Reply:
I started off structuring it the way you have it but then changed it because the suspended option is way more complicated than the active one and required more explanation. You certainly cleaned it up, but maybe too much. 
If a user is suspended whilst using Neo4j, they only lose the PUBLIC role if they have no other authorization. If they authorize against something else (e.g. LDAP) they will not lose the public role. This is to avoid breaking use cases where native users are being created and suspened to allow e.g. setting home database for that user when the log in with LDAP. The account would be suspended because we don't want to use the native credentials to sign in, but we do want to be able to store the home database somewhere.

Reply:
<img width="974" alt="Screenshot 2023-11-22 at 12 42 39" src="https://github.com/neo4j/docs-operations/assets/10127157/1036d423-6065-4013-a705-a05046593721">

Reply:
<img width="960" alt="Screenshot 2023-11-22 at 12 44 16" src="https://github.com/neo4j/docs-operations/assets/10127157/05ad503f-8761-4e01-b404-72ddb14d0e49">

Reply:
@mnd999, how about now?

Reply:
Any thoughts?

Reply:
Yeah, that works for me.

Reply:
Perfect! Thanks, I'll update the PR.